frontend/src/components/GddTransactionList.vue

```diff
@@ -41,6 +41,10 @@
               :decayStartBlock="decayStartBlock"
             />
           </template>
+
+          <template #TRANSACTION_LINK>
```

Comment:
The name `TRANSACTION_LINK` seems not good, since its more of a `TRANSACTION_LINK_LIST` or `TRANSACTION_LINKS` or something.
Please clarify with @Mogge

Reply:
Let us discuss how to name it best.

Reply:
In this case it is not a list. It is more a summary of all pending transaction links.

Reply:
If you change the TypeId, then it is virtually only the template in the GddTransactionLIst that is affected.